### DISCUSSION TITLE
Morocco IPTV list

### DISCUSSION THREAD

**Discussion by malekairmaroc7** [iptv-org/community#606] [Q&A] [answered]
Hello!

I noticed that the majority of channels in the IPTV list for Morocco don't belong there making it pretty messy.

These are the only channels which belong to Morocco:

• 2M Monde
• Al Aoula International and Laayyoune
• Al Maghribia
• Arrabiaa
• Assadissa
• Arryadia
• M24 TV
• Maghreb Arab Press
• Medi1TV Afrique, Arabic and Maghreb
• Tamazight
• TeleMaroc
• Chada TV

The other channels mainly belong to UAE and Egypt. Not sure why they are included. I searched other discussions but this one makes it clear.

Thank you!

**Comment by bedevlab** [comment]
this issue seems to appear in most of countries

**Comment by BellezaEmporium** [comment, marked as answer]
This isn't an issue, it is just because the other channels can be aired or seen in Morocco, either through satellite or through OTT/IPTV providers (in Morocco it could be Maroc Telecom, Inwi or any other).

We know about this, and if you only want the Moroccan channels, you can create the playlist in https://iptv-org.github.io, type in "country:MA" in the search bar, and select all channels which have an available stream.
